OPENNLP-999: Use JBake for website (thanks to @wcolen), new layout, and logo

kinow opened the pull request:
Thank you for contributing to Apache OpenNLP.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ x ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ x ] Does your PR title start with OPENNLP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ x ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ x ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn clean install at the root opennlp folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ x ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file in opennlp folder?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found in opennlp folder?

### For documentation related changes:
- [ x ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

jzonthemtn commented:
It looks nice!

![opennlp-website](https://cloud.githubusercontent.com/assets/13176962/25384379/1fe96262-298e-11e7-8e02-0150477fbf74.png)

smarthi commented on opennlp-site/src/main/jbake/content/team.ad:
```diff
@@ -0,0 +1,40 @@
+* Suneel Marthi (smarthi) (C-P)
+* Daniel Russ (druss) ( C )
+* Peter Thygesen (thygesen) ( C )
+* Koji Sekiguchi (koji) ( C )
```
Also add Jeffrey Zemerick (jzemerick) (C) to this list

kinow replied:
Oh, also forgot to add myself....

smarthi commented on opennlp-site/src/main/jbake/content/team.ad:
```diff
@@ -0,0 +1,40 @@
+* Chris Mattmann (mattmann) ( C )
+* Anastasija Mensikova (anastasijam) ( C )
+* Suneel Marthi (smarthi) (C-P)
+* Daniel Russ (druss) ( C )
```
Daniel Russ is now C-P

kottmann commented:
All those files should probably have AL headers when possible.

kinow commented:
Pull request updated and squashed.

Team members list updated.

Added headers to most files, except for assets (JQuery JS, CSS files, etc). Comments for AsciiDoc are /// and for FreeMaker are like HTML but start with <#--. These comments do not generate extra blank lines when consumed by the template parser.

Also updated the News article page that was not displaying correctly, and removed news/index.html that was never used (templates/news.ftl gets served instead).

Thanks!
Bruno

kinow commented:
Gitpubsub for websites allows static content sites being served from a branch, asf-site by default [1]. We could merge
pull request #182, then have a build job in Jenkins, with the following Maven build step:

```
# build just the web site module
mvn clean package -pl opennlp-site -am

# given using something like ghp-import could be complicated [2]

git add -A -f opennlp-site/target/opennlp-site/
git commit -m "Update site"
git subtree split --prefix opennlp-site/target/opennlp-site/ -b asf-site
git push -f origin asf-site:asf-site

```

Trigger the job whenever someone commits to opennlp-site (don't remember the names in Jenkins, but you can filter the commit...). That way that branch would always be up to date. I believe this would not automatically reflect opennlp.apache.org. But instead, gitpubsub would push the site to opennlp.staging.apache.org (not sure though).

[1] https://blogs.apache.org/infra/entry/git_based_websites_available
[2] https://github.com/davisp/ghp-import/blob/master/ghp_import.py
